lib/clouds/shared_functions.py

```diff
@@ -825,64 +825,9 @@ def populate_interface(self, obj_attr_list) :
                     except Exception, msg :
                         _fmsg = "Could not lookup interface " + iface + " for hostname " + hostname + " (probably bad /etc/hosts): " + str(msg)
                         raise CldOpsException(_fmsg, 1295)
-                    
-    @trace
-    def is_cloud_image_uuid(self, imageid) :
```

Comment:
Why are these checks necessary in the first place? Why does it matter that image IDs can be uuids or not? Is this specific to OpenStack? And if so, could we just rip out the whole thing and keep it contained to openstack?

Reply:
This is actually used by OpenStack, LXD and Docker, among others. In addition to it, I am using this function to "translate" the images names (on `VM_TEMPLATES : <MODEL>_CLOUDCONFIG]`) into image uuids. I would really like to keep it, unless you strongly object to it. I am willing to even go and put it on "technical debt" if you insist on removal.